frigate/config.py

```diff
@@ -266,6 +267,7 @@ class StationaryConfig(FrigateBaseModel):
 
 
 class DetectConfig(FrigateBaseModel):
+    autoconf: bool = Field(default=True, title="Auto detect height, width and fps.")
```

Comment:
This should default to false or maybe be baked in to a default value of detect and height. Otherwise existing users who had a width / height specifically set would have that config ignored

Reply:
> This should default to false or maybe be baked in to a default value of detect and height. Otherwise existing users who had a width / height specifically set would have that config ignored

make sense. I was changed this behaviour in [0505874](https://github.com/blakeblackshear/frigate/pull/6810/commits/050587478a15bc874afba6598a67d00acf78be6d), but I'm sure it can be made somehow easier

Reply:
What if the autoconf field was removed and width / height defaulted to None? Do stream detection when they are none

Reply:
> What if the autoconf field was removed and width / height defaulted to None? Do stream detection when they are none

I have no idea how it's possible to specify it in the configuration. Give an example :)

Reply:
Here is an example of a field with default value of None https://github.com/blakeblackshear/frigate/blob/3efa77f3022069a29c5b23f71162027687d823ae/frigate/config.py#L76

Reply:
> Here is an example of a field with default value of None
> 
> https://github.com/blakeblackshear/frigate/blob/3efa77f3022069a29c5b23f71162027687d823ae/frigate/config.py#L76

We have small miscommunication here. Are you talking about the case if this setting is removed from the code in the future?

Reply:
I'll try to be clear. What I'm suggesting is that the autoconf field does not need to exist. We can instead just make the width and height default to None and have it to when they are none autoconf should be done and if the width / height are set it shouldn't be done

Reply:
> I'll try to be clear. What I'm suggesting is that the autoconf field does not need to exist. We can instead just make the width and height default to None and have it to when they are none autoconf should be done and if the width / height are set it shouldn't be done

Understood. Looks more convenient than separate "autoconf" setting .